Run `examples/pytests` in CI so bundled example tests can't silently rot

### First Check

- [x] I added a very descriptive title here.
- [x] This is not a security issue.
- [x] This is not a Q&A. I am sure something is wrong with NiceGUI's CI setup.
- [x] I used the GitHub search to find a similar issue and came up empty.

### Example Code

```python
# examples/pytests/tests/test_ci_blindspot_demo.py
def test_deliberately_broken() -> None:
    assert False, 'this example is broken on purpose'
```

```console
# A) CI's command — `uv run pytest` from repo root (pyproject: testpaths = ["tests"])
$ python -m pytest --collect-only -q
907 tests collected in 0.51s
$ python -m pytest --collect-only -q | grep -c test_ci_blindspot_demo
0                      # the broken example is NEVER collected; CI stays green

# B) run the example dir directly — the failure is real
$ cd examples/pytests && python -m pytest tests/test_ci_blindspot_demo.py -q
E       assert False
tests/test_ci_blindspot_demo.py:2: AssertionError
FAILED tests/test_ci_blindspot_demo.py::test_deliberately_broken - assert False
1 failed in 0.02s
```

(Ran on a current `main` checkout; the demo file was removed afterwards.)

### Description

The `examples/pytests` project ships runnable pytest files as teaching material, but **CI never executes them**, so a broken example stays green — worse than no example, because it teaches the wrong thing and erodes trust in the docs.

- **Expected:** a broken test under `examples/pytests` should turn CI red.
- **Actual:** it is never collected. `pyproject.toml` sets `testpaths = ["tests"]`, and no workflow references `examples/` — CI's `uv run pytest` (`.github/workflows/_test.yml`) collects only `tests/` (907 tests above, 0 of them the example).

This is not hypothetical: it surfaced in #6015, where the bundled scope example actually failed (assertions ran against the whole page instead of the scoped card) and nothing caught it. @falkoschindler asked for this follow-up issue there.

**Possible fix:** add a CI step that runs `pytest` inside `examples/pytests` (it has its own `conftest.py`/fixtures), or fold example test dirs into the main collection where fixtures allow. Worth deciding: just `examples/pytests` (has real tests today) vs. a broader sweep of any `examples/**/test_*.py`.

### NiceGUI Version

`main` (`3.11.1.post29.dev0+7d14357c`) — this is a CI-workflow gap, not version-specific.

### Python Version

`3.13.11` — also not version-dependent.

### Browser

Other (N/A — this is a CI/tooling issue; no browser involved).

### Operating System

Other (reproduced on macOS, but the gap is OS-independent — CI runs on Linux).

### Additional Context

Follow-up to #6015. The concrete acceptance test: add a deliberately-broken example, confirm CI goes red, then confirm it passes once fixed.

--- a/.github/workflows/_test.yml
+++ b/.github/workflows/_test.yml
@@ -35,11 +35,17 @@ jobs:
       - name: ${{ matrix.python }} / ${{ matrix.scope }}
         if: matrix.scope == 'pytest'
         run: uv run pytest
+      - name: ${{ matrix.python }} / examples/pytests
+        if: matrix.scope == 'pytest'
+        run: uv run --project ../.. pytest
+        working-directory: examples/pytests
       - uses: actions/upload-artifact@v7
         if: matrix.scope == 'pytest' && failure()
         with:
           name: screenshots-${{ matrix.python }}
-          path: screenshots/**/*.failed.png
+          path: |
+            screenshots/**/*.failed.png
+            examples/pytests/screenshots/**/*.failed.png
           retention-days: 14
       - name: ${{ matrix.python }} / ${{ matrix.scope }}
         if: matrix.scope == 'startup'

